lib/bitcoin/lock/LockMonitor.ts

```diff
@@ -277,12 +296,8 @@ export default class LockMonitor {
     latestSavedLockInfo: SavedLockModel,
     desiredLockAmountInSatoshis: number): Promise<boolean> {
 
-    const currentBlockTime = await this.bitcoinClient.getCurrentBlockHeight();
-
-    console.info(`Current block: ${currentBlockTime}; Current lock's unlock block: ${currentValueTimeLock.unlockTransactionTime}`);
-
     // Just return if we're not close to expiry
-    if (currentValueTimeLock.unlockTransactionTime - currentBlockTime > 1) {
+    if (! (await this.isUnlockTimeReached(currentValueTimeLock.unlockTransactionTime))) {

```

Comment:
I feel like we discussed this already, but I can't remember what the conclusion was. Between the time a new lock is broadcasted and when it is actually on the chain, during that time, the node will lose its ability to write large volume right?

Reply:
Yes you are right. There may be a block (or two) when this node will not have any lock.